deployment/aws-python-storage-stack-automation/README.md

```diff
@@ -1,6 +1,6 @@
 # Automation using Lambda to Deploy FSS- Storage Stack 
 
-Automated process to deploy an FSS storage stack on each new S3 bucket using Lambda. The storage stack will be linked to the scanner stack previously defined.
+Automated process to deploy an FSS storage stack on each new S3 bucket using Lambda. The storage stack will be linked to the scanner stack previously defined. When S3 bucket is deleted the associated FSS Storage stack will be removed. 
```

Comment:
Automated process to deploy an FSS storage stack on each new S3 bucket using Lambda. The storage stack will be linked to the scanner stack previously defined. When the S3 bucket is deleted, the associated FSS Storage stack will be removed.

Reply:
Done